docs/scheduler/rescheduling.md

```diff
@@ -0,0 +1,46 @@
+<!--[metadata]>
++++
+title = "Docker Swarm recheduling"
+description = "Swarm rescheduling"
+keywords = ["docker, swarm, clustering, rescheduling"]
+[menu.main]
+parent="smn_workw_swarm"
+weight=5
++++
+<![end-metadata]-->
+
+# Rescheduling
+
+The Docker Swarm scheduler is able to detect node failure and
+restart it's container on another node.
+
+## Rescheduling policies
+
+The rescheduling policies are:
+
+* `on-node-faillure`
+* `off` (default if not specified)
+
+
+When you start a container, use the env var `reschedule` or the
+label `com.docker.swarm.reschedule-policy` to specify the policy to
+apply to the container.
+
+```
+# do not reschedule (default)
+$ docker run -d -e reschedule:off redis
+# or
+$ docker run -d -l com.docker.swarm.reschedule-policy=off redis
```

Comment:
I'm good with both too, I think it goes along with what we've always been doing with affinities and constraints.